Cargo.toml

```diff
@@ -206,17 +206,20 @@ sha1_smol = { opt-level = 3 }
 
 [profile.release]
 overflow-checks = false
-#lto = "fat"
-# this bloats files but assures destructors are called, important for tempfiles. One day I hope we
+lto = "thin"
```

Comment:
It seems to me that `lto = "thin"` here in the `release` profile is reasonable, but as noted above it does slow down builds a bit. In case you don't want it, you can accept this suggestion:

```suggestion
```

This is for convenience if that is the case, but not to advocate this change.

It might make sense to wait until benchmarking is done before adding this, though I believe it usually improves runtime performance--unlike going from thin to fat LTO, which is not always the case--and decisively determining that it is not worthwhile may involve benchmarking on quite a wide variety of architectures.

(This change would not affect the `release-opt` profile, because while that inherits from this, it has its own value here, setting it to `fat`.)